modules/hardware/nvidia-jetson-orin/ota-utils-fix.nix

```diff
@@ -14,6 +14,7 @@
   # mkAfter needed here so that we can be sure the overlay is after the overlay
   # included from jetpack-nixos. Otherwise it will just override the whole
   # nvidia-jetpack set.
+  # TODO remove when merged to upstream
```

Comment:
https://github.com/anduril/jetpack-nixos/pull/162

It's probably already fixed by this, if our flake.lock file has been bumped to new enough version